spec/models/widget/card_spec.rb

```diff
@@ -45,4 +45,36 @@
       expect(Widget::Card.body).not_to include(page_card)
     end
   end
+
+  context "when is not header card" do
+    it "link_url field is required" do
+      invalid_card = build(:widget_card, header: false, link_url: nil)
+
+      expect(invalid_card).not_to be_valid
+    end
+  end
+
+  context "when is header card" do
+    it "is not required if link_text is not provided" do
```

Comment:
I guess we mean `"link_url is not required if (...)` or `"is valid if neither link_text nor link_url are provided"` :thinking:.